chainer/training/extensions/plot_report.py

```diff
@@ -0,0 +1,143 @@
+import json
+import numpy
```

Comment:
Sorry for my confusing comments. `numpy` is a third party package, so it should be placed after this standard package part. Considering the alphabetical order, it's better to be placed right before `import six` :)